R/tribble.R

```diff
@@ -9,6 +9,9 @@
 #'
 #' @param ... Arguments specifying the structure of a `tibble`.
 #'   Variable names should be formulas, and may only appear before the data.
+#'   If a function needs to be applied to the data for a column (e.g. `factor`),
+#'   this should apear after the `~`, e.g. factor(column_A). Other arguments
+#'   cab be provided as needed.
```

Comment:
Actually, the last option you suggested looks clearer to me, and more in line with usual tidyverse semantics. The term `colA~.` translates directly to a column named `colA` with no transformation.